asserts/toolbelt/main.go

```diff
@@ -1,204 +0,0 @@
-// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
Is this file removal on purpose in this particular branch? it looks a bit out-of-place here, especially given that the description does not mention it.